_emails/us/delaware/newark.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Newark, DE
+permalink: "/newark_de"
```

Comment:
for permalinks use hyphens

`permalink: "/newark-de"`

Reply:
will do now

Reply:
done!